packages/edit-site/src/components/sidebar-dataviews/custom-dataviews-list.js

```diff
@@ -51,8 +51,7 @@ function RenameItemModalContent( { dataviewId, currentTitle, setIsRenaming } ) {
 		>
 			<VStack spacing="5">
 				<TextControl
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
Rename the custom view you just made.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/f4159579-de74-4d89-8a68-8bc3e432a64e" alt="Rename custom dataview, before" width="410">|<img src="https://github.com/user-attachments/assets/edc9b7ab-6dbc-4bc3-9cb0-ab6d1d6c5d84" alt="Rename custom dataview, after" width="411">|